database/startup_schema.js

```diff
@@ -8,7 +8,8 @@ const startupSchema = mongoose.Schema({
   startupSector: {type: String, required: true},
   email: {type: String, required: true},
   fundingStage: {type: String, required: true},
-  startupWebsite: {type: String, required: true },
+  startupWebsite: {type: String, required: true},
+  signUpReason: {type: String, required: true}
```

Comment:
I think if you need it for us as you said in your other comment, it should be requierd, no?
I'm happy to change it if you and @esraajb think it shouldn't.